test/kubemark/start-kubemark.sh

```diff
@@ -218,6 +218,13 @@ function create-and-upload-hollow-node-image {
 # Use bazel rule to create a docker image for hollow-node and upload
 # it to the appropriate docker container registry for the cloud provider.
 function create-and-upload-hollow-node-image-bazel {
+  # https://github.com/bazelbuild/rules_docker#authorization
+  # we need docker-credential-gcr to get authed properly
+  if ! [ -x "$(command -v docker-credential-gcr)" ]; then
+    gcloud components install docker-credential-gcr
```

Comment:
This might be fine for now, but we don't really want cloud-provider specific commands inside kubemark scripts. Kubemark setup currently is possible on non-gcp providers too. This was the reason why the docker push command earlier was refactored.

Reply:
good point, how about add this logic to kubetest before calling start-kubemark.sh so that I can check provider?